ActionGame/ActionGame/Script/Header/sc_Base.h

```diff
@@ -0,0 +1,50 @@
+
+#ifndef SC_BASE_H
+#define SC_BASE_H
+
+class SceneBase {
+public:
+	// �R���X�g���N�^
+	SceneBase();
+
+	// �f�X�g���N�^
+	~SceneBase();
+
+	virtual void SceneProcessing() = 0;
+	virtual void GameProcessing() = 0;
+	virtual void DrawProcessing() = 0;
```

Comment:
```suggestion
	virtual void Control() = 0;
	virtual void Draw() = 0;
```
といったメソッド名にすることが多いです。
いずれにせよ、動詞です。

Reply:
@taishi-matsumura 
180ed4c と d98b8b2 で修正しました。

一回 `Game` にしちゃったのを隠蔽するの忘れてました。